packages/trader/src/Modules/Trading/Components/Elements/__tests__/purchase-fieldset.spec.tsx

```diff
@@ -35,7 +48,8 @@ const default_mocked_props = {
     purchased_states_arr: [true, false],
     setPurchaseState: jest.fn(),
     type: '',
-} as unknown as React.ComponentProps<typeof PurchaseFieldset>;
+    is_beta_chart: false,
```

Comment:
@kate-deriv maybe it's better to not add this field at all because all mentions of is_beta_chart will be removed by @ahmad

Reply:
Yes, we will remove the beta flag soon. The card to remove it is already in QA.

Reply:
Until it's merged to master, TS gives error. As soon as changes will be released, I'll remove it

Reply:
![Uploading Screenshot 2023-11-22 at 09.30.15.png…]()

Reply:
<img width="924" alt="Screenshot 2023-11-22 at 09 30 15" src="https://github.com/binary-com/deriv-app/assets/121025168/9231b243-0bdf-42ed-9c03-2784a5a511f8">

Reply:
I've added a note to the CU card, that it should be merged only after world wide flutter chart release